virtool/oidc/api.py

```diff
@@ -0,0 +1,90 @@
+import os
+from urllib.parse import parse_qs
+
+import jwt
+from aiohttp.web import Request, Response
+from aiohttp.web_exceptions import HTTPFound
+
+from virtool.http.routes import Routes
+
+routes = Routes()
+
+
+@routes.get("/api/acquire_tokens")
```

Comment:
Use the `public` kwarg instead of storing a list of "unauthenticated routes".
```suggestion
@routes.get("/api/acquire_tokens", public=True)
```